cmd/tmsUpload.go

```diff
@@ -110,7 +111,10 @@ func runTmsUpload(config tmsUploadOptions, communicationInstance tms.Communicati
 		return fmt.Errorf("mta file %s not found", mtaPath)
 	}
 
-	description := config.CustomDescription
+	description := tms.DEFAULT_TR_DESCRIPTION
```

Comment:
This change is also part of PR #4134 (branch 'tms_integration'). Maybe it would make sense to base tms_integration branch on this branch and change the base of PR #4134 on this branch?